viime/app.py

```diff
@@ -59,6 +60,8 @@ def create_app(config=None):
     app.config['MAX_CONTENT_LENGTH'] = int(os.getenv('MAX_FILE_UPLOAD_SIZE', 5 * 1024 * 1024))
     app.config['OPENCPU_API_ROOT'] = os.getenv('OPENCPU_API_ROOT')
 
+    CORS(app)
```

Comment:
```suggestion
    CORS(app, origins="*")
```
The unconfigured Flask CORS plugin will actually block all cross origin requests by default. You need to specify somewhere that all origins are acceptable.

@jbeezley should the allowed origins go in the `.gunicorn_env` instead?